evap/results/templates/distribution_bar.html

```diff
@@ -0,0 +1,11 @@
+{% load results_templatetags %}
+
+{% spaceless %}
+<div class="distribution-bar">
+    {% for answer, value in distribution.items %}
+        {% if value != 0 %}
+            <div class="vote-bg-{{ answer }}{% if participants_warning %} participants-warning{% endif %}" style="width: {% widthratio value distribution.values|sum 100 %}%;">&nbsp;</div>
```

Comment:
that sum should now always be one, right?

Reply:
right